lib/membrane/video_compositor/download_compositor.ex

```diff
@@ -16,8 +16,12 @@ defmodule Mix.Tasks.DownloadCompositor do
 
     path = File.cwd!() |> Path.join("video_compositor_app/#{vc_architecture}")
 
-    File.mkdir_p!(path)
-    _wget_res = "wget -nc #{url} -O - | tar -xvz -C #{path}" |> String.to_charlist() |> :os.cmd()
+    unless File.exists?(path) do
+      File.mkdir_p!(path)
+
+      _wget_res =
+        "wget -nc #{url} -O - | tar -xvz -C #{path}" |> String.to_charlist() |> :os.cmd()
```

Comment:
1) Use `System.cmd` instead of `:os.cmd`
2) Don't ignore return value of this command